pom.xml

```diff
@@ -36,7 +36,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.10.5.1</version>
+            <version>2.13.5</version>
```

Comment:
Hey @rishabhbits038 let's update to the latest version

Reply:
Will update this to 2.16.1

Reply:
or define it to the property coming from the parent. I think it is `<jackson.version>`

Reply:
updated.